README.md

```diff
@@ -42,18 +42,25 @@ Create file _/etc/goodwe.conf_:
 
 ```
 [inverter]
-loglevel = INFO
-pollinterval = 2500
-vendorId = 0084
-modelId = 0041
-logfile = /var/log/goodwe.log
+#loglevel = DEBUG
+#pollinterval = 2500
+#vendorId = 0084
+#modelId = 0041
+#logfile = /var/log/goodwe.log
 
 [mqtt]
-server = <MQTT Server IP>
-port = 1883
-topic = goodwe
-clientid = <unique MQTT client id>
+#server = localhost
+#port = 1883
+#topic = goodwe
+#clientid = goodwe-usb
+#username = 
+#password = mypassword
+
 ```
+Almost all configuration parameters have sensible defaults and are commented out. The values shown are the defaults. If you ned to change a 
```

Comment:
Please correct the spelling of "If you ned to change a"

Reply:
Overlooked that error. Is corrected

Reply:
Thanks